crates/rpc/src/pathfinder/methods/get_proof.rs

```diff
@@ -116,6 +138,14 @@ pub async fn get_proof(
     context: RpcContext,
     input: GetProofInput,
 ) -> Result<GetProofOutput, GetProofError> {
+    const MAX_KEYS: usize = 100;
```

Comment:
Just wondering if it makes any sense to make it configurable? I am sure sooner or later someone is going to be hit by this constraint :)

Reply:
It does; but that would be more work :D For now people can make multiple requests. Hmm. Although maybe.. we should just get it done now already..